bus/bus.go

```diff
@@ -334,7 +323,7 @@ type Bus struct {
 }
 
 // New returns a new Bus
-func New(cfg config.Bus, masterKey [32]byte, am AlertManager, wm WebhooksManager, cm ChainManager, s Syncer, w Wallet, store Store, explorerURL string, l *zap.Logger) (_ *Bus, err error) {
+func New(cfg config.Bus, masterKey [32]byte, am AlertManager, cm ChainManager, s Syncer, w Wallet, store Store, explorerURL string, l *zap.Logger) (_ *Bus, err error) {
```

Comment:
After merging this we need to update `cluster` since that will break it

Reply:
Heyyy, you membered!